package.json

```
@@ -25,7 +25,7 @@
     "node": "14.x"
   },
   "dependencies": {
-    "express": "^4.18.0",
+    "express": "^4.18.2",
```

The package.json must remain untouched, please revert this change
```suggestion
    "express": "^4.18.0",
```